src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/wrapper/CK_MECHANISM.java

```diff
@@ -193,11 +205,12 @@ public String toString() {
 
         sb.append(Constants.INDENT);
         sb.append("mechanism: ");
-        sb.append(mechanism);
+        sb.append(Functions.getMechanismName(mechanism));
         sb.append(Constants.NEWLINE);
 
         sb.append(Constants.INDENT);
-        sb.append("pParameter: ");
+        sb.append("pParameter:");
+        sb.append(Constants.NEWLINE);
```

Comment:
Is this intended? It seems NEWLINE is not added beforehand for other fields.

Reply:
We found it more clear to start in a new line when showing the inner parameters. The reason is that inner parameters can be a structure (as in the PBE case) which has its own "member-name:" and new lines. It looked a bit odd when we had for example "    pParameter:    pInitVector:" in the first line and the rest of the inner structure below.

Reply:
I see. Sure.